docs/reference/plugins.md

```diff
@@ -43,14 +44,92 @@ There are some pitfalls:
 * do not attempt to rebuild a module and copy it over a loaded plugin, that
   will crash skipper immediately...
 
+## Use a plugin
+
+In this example we use a geoip database, that you need to find and download.
+We expect that you did a `git clone git@github.com:zalando/skipper.git` and
+entered the directory.
+
+Build skipper:
+
+```
+make skipper
```

Comment:
For consistency, this line should start with `%` as well

Reply:
fixed